temporalcli/commands.workflow_trace.go

```diff
@@ -0,0 +1,133 @@
+package temporalcli
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+	"time"
+
+	"github.com/temporalio/cli/temporalcli/internal/tracer"
+
+	"go.temporal.io/api/enums/v1"
+
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"go.temporal.io/sdk/client"
+)
+
+var foldFlag = map[string]enums.WorkflowExecutionStatus{
```

Comment:
```suggestion
var workflowTraceFoldFlags = map[string]enums.WorkflowExecutionStatus{
```

This package is shared by many commands, need to qualify global names